scheduling/allocator.rb

```diff
@@ -570,6 +570,19 @@ def self.allocate_spdk_installation(spdk_installations)
       rand_choice.id
     end
 
+    def self.allocate_vhost_block_backend(backends)
+      total_weight = backends.sum(&:allocation_weight)
+      fail "Total weight of all eligible vhost_block_backends shouldn't be zero." if total_weight == 0
+
+      rand_point = rand(0...total_weight)
```

Comment:
```suggestion
      rand_point = rand(total_weight)
```

No need for a range here.